.idea/misc.xml

```diff
@@ -21,22 +21,49 @@
     <option name="myDefaultNotNull" value="org.checkerframework.checker.nullness.qual.NonNull" />
     <option name="myNullables">
       <value>
-        <list size="3">
-          <item index="1" class="java.lang.String" itemvalue="org.checkerframework.checker.nullness.qual.Nullable" />
-          <item index="2" class="java.lang.String" itemvalue="org.checkerframework.checker.nullness.compatqual.NullableDecl" />
-          <item index="3" class="java.lang.String" itemvalue="org.checkerframework.checker.nullness.compatqual.NullableType" />
+        <list size="14">
```

Comment:
Please make sure this file looks more like the [original one](https://github.com/SpineEventEngine/config/blob/master/.idea/misc.xml).

I don't think we really need all these `@Nullable`-related annotations.